library/ssl_tls13_generic.c

```diff
@@ -1061,15 +1055,15 @@ static int ssl_tls13_write_certificate_verify_body( mbedtls_ssl_context *ssl,
     p += 2;
 
     /* Hash verify buffer with indicated hash function */
-    md_info = mbedtls_md_info_from_type( md_alg );
-    if( md_info == NULL )
-        return( MBEDTLS_ERR_SSL_INTERNAL_ERROR );
+    psa_algorithm = mbedtls_psa_translate_md( md_alg );
 
-    ret = mbedtls_md( md_info, verify_buffer, verify_buffer_len, verify_hash );
-    if( ret != 0 )
+    if(psa_hash_compute( psa_algorithm,
```

Comment:
Style: `if( psa_hash_compute(` (space). Sorry, I know our style is unusual and we want to change it at one point, but in the meantime, we should be consistent about it.